dynamite_nsm/services/kibana/package/manager.py

```diff
@@ -0,0 +1,784 @@
+                    kibana_objects_file.close()
+                    installation_statuses.append(self._process_package_installation_results(package, result))
+            # Should we remove this and ONLY install validatable packages?
+            elif filetype in ('application/json', 'text/plain') and filepath.endswith('ndjson'):
+                with open(filepath, 'r') as kibana_objects_file:
+                    result = self.import_kibana_saved_objects(username=username, password=password,
+                                                    kibana_objects_file=kibana_objects_file)
+                    installation_statuses.append(self._process_package_installation_results(package, result))
+            else:
+                self.logger.error("Files must be one of: .ndjson, .json, .tar.xz, .tar.gz")
+                # TODO raise exception
+        
+        if not all(installation_statuses):
+            print(f"\r\n{package.manifest.name} installation failed.")
+            if not self.verbose:
+                print("Use --verbose flag to see more error detail.\n")
+        else:
+            package.register()
+            print(f"\r\n{package.manifest.name} installation succeeded!")
+
+    def list(self, username: Optional[str] = None, password: Optional[str] = None, pretty: Optional[bool] = False):
+        """List packages currently installed for this instance
+
+        Args:
+            username (Optional[str], optional): kibana auth username. Defaults to None.
+            password (Optional[str], optional): kibana auth passwd. Defaults to None.
+        """
+        packages = Package.search_installed_packages()
+        if not packages:
+            print("Could not find any installed packages")
+            return None
+        
+        if pretty:
+            print("\r\nInstalled Packages:\n")
+            headers = ["Package Name", "Package ID", "Description", "Author", "Objects Within", "Total Objects"]
+            table = []
+            for package in packages:
+                row = []
+                total = 0
+                row.append(package.manifest.name)
+                row.append(package.id)
+                row.append(package.manifest.description)
+                row.append(package.manifest.author)
+                objlines = []
+                for obj in package.installed_objects:
+                    objlines.append(f"[{obj.object_type}] - {obj.title}")
+                    total += 1
+                row.append("\n".join(sorted(objlines)))
+                row.append(total)
+                table.append(row)
+            print(tabulate(table, headers=headers, tablefmt="fancy_grid"))
+        else:
+            data = []
+            for package in packages:
+                data.append(package.es_input())
+            print(data)
+        
+    def list_saved_objects(self, username: Optional[str] = None, password: Optional[str] = None,
```

Comment:
Let's return a string here instead of printing it, defaults to json. The cmd modules enable this functionality automatically for `entry_methods` in single/multi resp interfaces

Here's a reference implementation:  https://github.com/DynamiteAI/dynamite-nsm/blob/feature/configmanager-tests/dynamite_nsm/services/base/process.py#L100-L166